<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeSelector</class>
 <widget class="QWidget" name="TreeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File selection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/windowicon.png</normaloff>icons/windowicon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>583</width>
        <height>620</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="wdgActions" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="boxActions">
            <item>
             <layout class="QHBoxLayout" name="boxContainer">
              <item>
               <widget class="QComboBox" name="cbRootNode">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="pathlist">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="flow">
                 <enum>QListView::LeftToRight</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="dragEnabled">
               <bool>false</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>96</width>
                <height>96</height>
               </size>
              </property>
              <property name="movement">
               <enum>QListView::Snap</enum>
              </property>
              <property name="layoutMode">
               <enum>QListView::Batched</enum>
              </property>
              <property name="gridSize">
               <size>
                <width>128</width>
                <height>128</height>
               </size>
              </property>
              <property name="viewMode">
               <enum>QListView::IconMode</enum>
              </property>
              <property name="uniformItemSizes">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddSelected">
         <property name="text">
          <string>Add to selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSelected">
         <property name="text">
          <string>Selected:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listSelected">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelect">
         <property name="text">
          <string>Apply</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/actions/relationalselect.png</normaloff>icons/actions/relationalselect.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="boxUpload"/>
   </item>
  </layout>
  <zorder>scrollArea</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
